test/providers/google.test.ts

```diff
@@ -36,4 +36,33 @@ describe('google', () => {
     expect(resolvedStyles).toMatchObject(styles)
     expect(resolvedWeights).toMatchObject(weights)
   })
+
+  it('supports variable axes', async () => {
+    const unifont = await createUnifont([providers.google({
+      experimental: {
+        variableAxis: {
+          Recursive: {
+            slnt: ['-15..0'],
+            CASL: ['0..1'],
+            CRSV: ['0..1'],
+            MONO: ['0..1'],
```

Comment:
would it make more sense for the format to be:

```suggestion
            slnt: ['-15', '0'],
            CASL: ['0', '1'],
            CRSV: ['0', '1'],
            MONO: ['0', '1'],
```

it feels like the `..` joining is an implementation detail for google that we can add in...

Reply:
@danielroe Resolved. Sorry for the delay!